resources/build/offline-help-style-spec.txt

```diff
@@ -0,0 +1,258 @@
+ * CSS RESET
+ * ***********************************/
+
+/* http://meyerweb.com/eric/tools/css/reset/
+   v2.0 | 20110126
+   License: none (public domain)
+*/
+
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed,
+figure, figcaption, footer, header, hgroup,
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure,
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+  padding-left: 10px;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+}
+
+tbody {
+    display: table-row-group;
+    vertical-align: middle;
+}
+
+table td{
+	padding: 4px;
+}
+
+html{
+		font-family: 'Cabin', sans-serif;
+		color: #333;
+}
+
+html p,
+html dd,
+html li {
+  font-family: 'Calibri', sans-serif;
+  font-size: 14pt;
+}
+
+body {
+	line-height: inherit;
+}
+
+/**************************************
+ * Page Layout CSS
+ * ***********************************/
+
+html{
+    cursor: default;
+    background: #fff;
+}
+
+:not(div.interface-hierarchy) blockquote {
+  margin-left: 32px;
+  margin-bottom: 16px;
+  background: rgba(242,242,242,0.9);
+  padding: 4px;
+  border-radius: 4px;
+}
+
+h1{
+	text-align: left;
+	font-size: 28pt;
+	padding: 40px 10px 10px 10px;
+}
+
+h2{
+	font-size: 20pt;
+	line-height: 1.4;
+	padding: 32px 10px 10px 10px;
+}
+
+h3{
+	font-size: 16pt;
+	font-weight: 600;
+	padding: 16px 10px 10px 10px;
+}
+
+h3, h2, h1 {
+  color: #B92034;
+  margin-top: 10px;
+}
+
+h1 a, h2 a, h3 a {
+  color: #B92034;
+}
+
+h1 a:visited, h2 a:visited, h3 a:visited {
+  color: #B92034;
+}
+
+h4{
+  font-size: 14pt;
+  font-weight: 600;
+  padding: 4px 10px 4px 10px;
+}
+
+p, dl, .itemizedlist {
+	padding: 0px 10px 10px 10px;
+	line-height: 1.4;
+	font-size: 14pt;
+}
+
+p a{
+	color: #B92034;
+	text-decoration: underline;
+	cursor: pointer;
+	font-weight: 600;
+	line-height: 1.4;
+}
+
+li{
+	margin-left: 30px;
+	list-style: disc;
+	list-style-position: outside;
+	line-height: 1.4;
+  font-size: 14pt;
+}
+
+img.center{
+	margin-left: auto;
+	margin-right: auto;
+	display: block;
+}
+
+img.inline {
+  margin: 0;
+}
+
+.itemizedlist li{
+	list-style-position: outside;
+}
+
+li {
+  margin-bottom: 12px;
+}
+
+/* TODO: deprecate this since prismjs handles it. This also conflicts with the line-numbers plugin */
```

Comment:
This looks good.  Can we also add styling for `<kbd>`  keycaps like how
https://help.keyman.com/products/android/15.0/basic/switching-between-keyboards

mentions "Ctrl" and "Tab" keys?

Reply:
Ah, so that _is_ something that can come through.  Looks like I may have pruned a bit too aggressively.

Note that you may want to tweak things a little bit:

From the actual help site:

![Screen Shot 2021-05-27 at 8 56 07 AM](https://user-images.githubusercontent.com/25213402/119753785-8ab8c180-bec9-11eb-8174-1e50e8ea0417.png)

The offline results:

![Screen Shot 2021-05-27 at 8 56 20 AM](https://user-images.githubusercontent.com/25213402/119753815-9ad0a100-bec9-11eb-813e-11b193b77a1b.png)

Looks like the custom font selected for the 'key caps' may not be preinstalled on Mac, and I can't immediately speak for Android.